projects/1.16/assets/dense-ores-reborn/denseoresreborn/lang/zh_cn.json

```
@@ -0,0 +1,22 @@
+{
+	"block.denseoresreborn.dense_iron_ore": "致密铁矿石",
+	"block.denseoresreborn.dense_emerald_ore": "致密绿宝石矿石",
+	"block.dense_ores_reborn.dense_redstone_ore": "致密红石矿石",
+	"block.denseoresreborn.dense_coal_ore": "致密煤矿石",
+	"block.denseoresreborn.dense_gold_ore": "致密金矿石",
+	"block.dense_ores_reborn.dense_emerald_ore": "致密绿宝石矿石",
+	"block.dense_ores_reborn.dense_lapis_lazuli_ore": "致密青金石矿石",
+	"block.dense_ores_reborn.dense_nether_gold_ore": "致密下界金矿石",
+	"block.dense_ores_reborn.dense_ancient_debris_ore": "致密远古残骸",
+	"block.dense_ores_reborn.dense_gold_ore": "致密金矿石",
+	"block.dense_ores_reborn.dense_diamond_ore": "致密钻石矿石",
+	"block.denseoresreborn.dense_nether_gold_ore": "致密下界金矿石",
+	"block.denseoresreborn.dense_quartz_ore": "致密下界石英矿石",
+	"block.dense_ores_reborn.dense_iron_ore": "致密铁矿石",
+	"block.dense_ores_reborn.dense_coal_ore": "致密煤矿石",
+	"block.denseoresreborn.dense_redstone_ore": "致密红石矿石",
+	"block.denseoresreborn.dense_diamond_ore": "致密钻石矿石",
+	"block.denseoresreborn.dense_ancient_debris_ore": "致密远古残骸矿石",
```

```suggestion
	"block.denseoresreborn.dense_ancient_debris_ore": "致密远古残骸",
```
个人认为此处不需强调矿石，加上你前文也这么翻译